hvantk/ptm/plot.py

```
@@ -84,34 +84,69 @@ def plot_landscape_summary(
+        f"OR={result.enrichment_odds_ratio:.2f} "
+        f"({result.enrichment_ci_low:.2f}–{ci_hi_str})\n"
+        f"p={result.enrichment_p_value:.2e}",
+        transform=ax1.transAxes, ha="right", va="top", fontsize=9,
         bbox=dict(boxstyle="round,pad=0.3", facecolor="wheat", alpha=0.5),
     )
+    ax1.grid(axis="y", linestyle="--", alpha=0.4)
+
+    # Right panel: % P/LP per stratum
+    bar_colors = [PTM_COLORS["ptm_site"], PTM_COLORS["proximal"], PTM_COLORS["non_ptm"]]
+    bars = ax2.bar(x, pct_plp, color=bar_colors, edgecolor="black", linewidth=0.5)
+    for bar, pct, tot in zip(bars, pct_plp, totals):
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)